src/Control/Eff/Internal.hs

```diff
@@ -257,7 +253,10 @@ raise = loop
 -- | Lifting: emulating monad transformers
 newtype Lift m a = Lift (m a)
 
--- | We make the Lift layer to be unique, using SetMember
+-- | lift a operation of type `m a` into the `Eff` monad.
```

Comment:
@power-fungus since we're focusing on improving, any ideas on how we can make this explanation less circular? what does "lift" in the context of "lift an operation" mean?

nitpick: typo, `s/a operation/an operation/`

Reply:
@suhailshergill maybe by calling it "embedding" will make it clearer